manubot/pandoc/tests/test_bibliography.py

```diff
@@ -43,7 +43,7 @@ def test_load_bibliography_from_text(path):
     """
     https://zbib.org/c7f95cdef6d6409c92ffde24d519435d
     """
-    text = path.read_text()
+    text = path.read_text(encoding='utf-8')
```

Comment:
Let's use `utf-8-sig` as per discussion in https://github.com/manubot/manubot/pull/127